src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php

```diff
@@ -45,6 +45,10 @@ public function handle($request, Closure $next)
                 return $next($request);
             }
 
+            if (isset($data['except']) && in_array($request->getRequestUri(), (array) $data['except'])) {
```

Comment:
I would personally expect to have exactly the same feature as `\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken::inExceptArray`.

Reply:
Great idea, thank you! This is even better.